cmd/controlplane/controller.go

```diff
@@ -106,7 +106,7 @@ func newControllerCommand() *cobra.Command {
 			}
 
 			var argocdMgr manager.Manager
-			{
+			if types.MustParseBool(os.GetEnv("ARGOCD_INTEGRATION_ENABLED", "true")) {
```

Comment:
This is a spot where we might try to make an inference from the presence or absence of Rollouts CRDs.